Misc/NEWS.d/next/Library/2018-09-17-19-11-17.bpo-34689.RtyOE3.rst

```diff
@@ -0,0 +1,2 @@
+Split strings prior to parsing in :func:`sysconfig._parse_makefile` so it
```

Comment:
`sysconfig._parse_makefile` is not a documented function, so I don't think the `func` role should be used here.